addons/account/models/mail_message.py

```diff
@@ -78,6 +89,17 @@ def _compute_account_audit_log_preview(self):
             )
             message.account_audit_log_preview = audit_log_preview
 
+    def _search_account_audit_log_preview(self, operator, value):
+        if operator != 'ilike' or not isinstance(value, str):
+            raise UserError(self.env._('Operation not supported'))
```

Comment:
The correct way to allow finding another valid operator (i.e. enabling `not ilike`)
```suggestion
            return NotImplemented
```